src/layouts/Base.astro

```diff
@@ -33,71 +73,130 @@ const pathname = Astro.url.pathname.replace(/\/$/, "")
-				</p>
+		<Header />
+
+		<div class="noise-container overflow-x-hidden">
+			<div class="noise"></div>
+
+			<div class="relative">
+				<div
+					class="noise-underlay bg-grid absolute inset-0 mask-linear-gradient-to-b"
+				>
+				</div>
+				<div class="container">
+					<Hero />
+				</div>
+			</div>
+
+			<div
+				class="noise-underlay absolute left-0 top-[350px] h-[600px] w-[calc(max(50vw,300px))] -translate-y-1/2 translate-x-[-60%] rounded-full bg-blue-purple-gradient opacity-75 mask-radial-gradient"
+			>
+			</div>
+			<div
+				class="noise-underlay absolute right-0 top-[350px] h-[600px] w-[calc(max(50vw,300px))] -translate-y-1/2 translate-x-[60%] rounded-full bg-orange-yellow-gradient opacity-75 mask-radial-gradient"
+			>
 			</div>
-		</PageTitle>
-
-		<main
-			class="mx-auto flex max-w-screen-xl flex-col px-4 pt-8 sm:px-6 sm:pt-12 lg:px-8 lg:pt-16"
-		>
-			<ul class="templates mb-24 flex flex-col gap-24">
-				{
-					groups.map(([category, examples]) => {
-						return (
-							<li class="flex flex-col gap-8">
-								<h2
-									class:list={[
-										"text-2xl",
-										{ "sr-only": category === TOP_SECTION },
-									]}
-								>
-									{category}
-								</h2>
-								<ul class="grid gap-4 sm:grid-cols-2 lg:grid-cols-3 lg:gap-6">
-									{examples.map(
-										(
-											example: astroHTML.JSX.IntrinsicAttributes & CardProps,
-										) => (
-											<li>
-												<Card {...example} />
-											</li>
-										),
-									)}
-									{category === TOP_SECTION && <Outlinks />}
-								</ul>
+
+			<main class="container">
+				<nav class="flex flex-wrap border-b border-astro-gray-500">
+					{
+						[...Astro.props.examples.values()].map((group) => (
+							<a
+								href={joinPath(Astro.props.basePath, group.slug)}
+								data-active={group.slug === currentExample || undefined}
+								class="inline-block rounded-t-lg from-astro-blue/50 to-emerald-300/50 px-4 py-2 text-sm transition hover:backdrop-brightness-50 data-[active]:bg-gradient-to-tr"
+							>
+								{group.title}
+							</a>
+						))
+					}
+				</nav>
+
+				<ul
+					class="mb-24 mt-8 grid gap-4 sm:grid-cols-2 lg:grid-cols-3 lg:gap-6"
+				>
+					{
+						exampleGroup.items.map((example) => (
+							<li>
+								<RepoCard {...example} />
 							</li>
-						)
-					})
-				}
-			</ul>
-		</main>
+						))
+					}
+				</ul>
+
+				<section class="flex flex-col items-center">
+					<img src={houstonHappy} alt="" />
+					<h2 class="heading-2 text-center">Houston, we have resources</h2>
+					<div
+						class="my-16 flex w-full flex-wrap gap-4 children:flex-1 children:basis-72"
+					>
+						<div class="border border-astro-gray-400 bg-red-pink-gradient">
+							<ResourceCard
+								title="Learn more\nin our docs"
+								description="Thanks to our wonderful contributors, our docs are the best place to learn, troubleshoot, and explore Astro."
+								link={{ text: "Start learning", href: "/" }}
```

Comment:
All these cards have an `href` of `/`. Should presumably point to the relevant resources?